src/docs/product/integrations/integration-platform/webhooks/installation.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Installation & Uninstallation
+sidebar_order: 1
+description: "Learn more about Sentry's Integration Platform installation and uninstallation webhooks."
+---
+
+A [public](/product/integrations/integration-platform/#installation) Sentry integration will receive installation and uninstallation webhooks.
```

Comment:
The hyper link for `public` looks a little wrong, and I think it should surround the words `Sentry integration` as well